onyo/lib/onyo.py

```diff
@@ -663,7 +663,7 @@ def _init_sanitize(directory: Union[Path, str]) -> Path:
     #
     def mkdir(self, directories: Union[Iterable[Union[Path, str]], Path, str]) -> None:
         """
-        Create ``directory``\(s). Intermediate directories will be created as
+        Create ``directory``/(s). Intermediate directories will be created as
```

Comment:
It renders the same in readthedocs as the previous version, by neither rendering the `/` or `\`. It just lists "Create `directory`(s)". I also don't find anything in the sphinx docs that assigns a function to `\` in this context., but perhaps I missed something. @TobiasKadelka do you have any comments on this?

Reply:
No I have no idea about how readthedocs and sphinx and their rendering works, which is the reason I ignored these so far, sorry!

Reply:
This has now been addressed. The `Repo.mkdir()` docstring is not used in the docs, whereas the (identical) `commands.mkdir()` is. Removing the `\` creates an error with sphinx, because sphinx expects \`` to be closed with another \``, but gets a \``(s) instead. To solve both the sphinx error and Python deprecation warning, I've replaced them all with `\\`'s. I've extended this fix to the `commands` section.